generated-structs.go

```diff
@@ -1,3929 +1,1706 @@
-	/// for pages to start loading. You can opt out of waiting via setting this flag. You
-	/// would only need this option in the exceptional cases such as navigating to inaccessible
-	/// pages. Defaults to <c>false</c>.
-	/// </para>
-	/// </summary>
+	// Actions that initiate navigations are waiting for these navigations to happen and for pages to start loading. You can opt out of waiting via setting this flag. You would only need this option in the exceptional cases such as navigating to inaccessible pages. Defaults to `false`.
 	NoWaitAfter *bool `json:"noWaitAfter"`
-	/// <summary>
-	/// <para>
-	/// A point to use relative to the top-left corner of element padding box. If not specified,
-	/// uses some visible point of the element.
-	/// </para>
-	/// </summary>
+	// A point to use relative to the top-left corner of element padding box. If not specified, uses some visible point of the element.
 	Position *FrameDblclickOptionsPosition `json:"position"`
-	/// <summary>
-	/// <para>
-	/// Maximum time in milliseconds, defaults to 30 seconds, pass <c>0</c> to disable timeout.
-	/// The default value can be changed by using the <see cref="BrowserContext.SetDefaultTimeout"/>
-	/// or <see cref="Page.SetDefaultTimeout"/> methods.
-	/// </para>
-	/// </summary>
-	Timeout *float64 `json:"timeout"`
-	/// <summary>
-	/// <para>
-	/// When set, this method only performs the <a href="./actionability.md">actionability</a>
-	/// checks and skips the action. Defaults to <c>false</c>. Useful to wait until the
-	/// element is ready for the action without performing it.
-	/// </para>
-	/// </summary>
+	// Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the BrowserContext.SetDefaultTimeout or Page.SetDefaultTimeout methods.
+	Timeout *float64 `json:"timeout"`
+	// When set, this method only performs the [actionability](./actionability.md) checks and skips the action. Defaults to `false`. Useful to wait until the element is ready for the action without performing it.
 	Trial *bool `json:"trial"`
 }
 type FrameDispatchEventOptions struct {
-	/// <summary>
-	/// <para>
-	/// Maximum time in milliseconds, defaults to 30 seconds, pass <c>0</c> to disable timeout.
-	/// The default value can be changed by using the <see cref="BrowserContext.SetDefaultTimeout"/>
-	/// or <see cref="Page.SetDefaultTimeout"/> methods.
-	/// </para>
-	/// </summary>
+	// Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the BrowserContext.SetDefaultTimeout or Page.SetDefaultTimeout methods.
 	Timeout *float64 `json:"timeout"`
 }
 type FrameDragAndDropOptions struct {
-	/// <summary>
-	/// <para>
-	/// Whether to bypass the <a href="./actionability.md">actionability</a> checks. Defaults
-	/// to <c>false</c>.
-	/// </para>
-	/// </summary>
+	// Whether to bypass the [actionability](./actionability.md) checks. Defaults to `false`.
```

Comment:
would be cool if we could resolve them to the official documentation, I think there are hooks in the markdown class for that.

Reply:
Nope even in python iit is same https://github.com/microsoft/playwright-python/blob/f57cc82814223bb66c6473b411fe3df4f5394000/playwright/async_api/_generated.py#L1474